bin/deferred_work_manager

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+require "tom_queue"
+
+module TomQueue
+  class Daemon
+    include LoggingHelper
+
+    def run(argv = ARGV)
+      return unless load_rails!(".")
+
+      deferred_manager = DeferredWorkManager.new
+      deferred_manager.start
+    end
+
+    # Load Rails environment
+    #
+    # Returns true/false indicate whether Rails is loaded or not
+    def load_rails!(path)
+      rails_path = File.expand_path(File.join(path, 'config/environment.rb'))
+      if File.exists?(rails_path)
+        info "Loading rails from `#{rails_path}`"
+
+        ENV['RACK_ENV'] ||= ENV['RAILS_ENV'] ||= 'development'
+        require rails_path
+        ::Rails.application.eager_load!
+        true
+      else
+        false
+      end
```

Comment:
Would it be better to have it exit noisily if rails isn't available for whatever reason? Currently if we call it from the wrong directory, then it exits silently (& successfully!).

Could just try loading rails regardless of the file existing, and then ruby will set the process exit code to non-zero and whinge about `Rails` not being defined.